src/Main.java

```diff
@@ -2,27 +2,52 @@
 import java.util.*;
 import java.util.stream.IntStream;
 
+class User {
+    private String name;
+    private int age;
+
+    public User(String name, int age) {
+        this.name = name;
+        this.age = age;
+    }
+
+    public String getName() {
+        return name;
+    }
+
+    public int getAge() {
+        return age;
+    }
+
+    @Override
+    public String toString() {
+        return "Name: " + name + ", Age: " + age;
+    }
+}
+
 public class Main {
     public static void main(String[] args) {
         Scanner scanner = new Scanner(System.in);
         System.out.print("Enter the number of users: ");
         int numberUser = scanner.nextInt();
         scanner.nextLine();
-        List<String> userNames = new ArrayList<>();
-        List<Integer> userAges = new ArrayList<>();
+
+        List<User> users = new ArrayList<>();
 
         IntStream.range(0, numberUser).forEach(i -> {
```

Comment:
Why do u use here IntStream.range()? What you expected form this operation?

Reply:
I use it to create a sequence of numbers in a specified range. In this case it is the number of users.  I want to receive the number of users to check the limit